admin-dev/themes/new-theme/package.json

```diff
@@ -51,35 +51,38 @@
     "karma-jquery": "^0.2.2",
     "karma-mocha": "^1.3.0",
     "karma-phantomjs-launcher": "^1.0.4",
-    "karma-phantomjs-shim": "^1.4.0",
-    "karma-sinon-chai": "^1.3.1",
+    "karma-phantomjs-shim": "^1.5.0",
+    "karma-sinon-chai": "^1.3.3",
     "karma-spec-reporter": "0.0.31",
-    "karma-webpack": "^2.0.4",
+    "karma-webpack": "^2.0.8",
     "loader-utils": "^1.0.3",
     "lodash": "^4.17.4",
     "magnific-popup": "^1.1.0",
-    "mobile-detect": "^1.3.5",
-    "mocha": "^3.4.1",
-    "moment": "^2.19.1",
-    "node-sass": "^4.5.0",
+    "mobile-detect": "^1.4.0",
+    "mocha": "^3.5.3",
+    "moment": "^2.19.4",
+    "node-sass": "^4.7.2",
     "path": "^0.12.7",
     "postcss-cssnext": "^2.10.0",
     "postcss-loader": "^1.3.3",
     "prestakit": "^1.0.0-beta-3",
     "sass-loader": "^6.0.6",
     "sinon": "^2.4.1",
-    "sinon-chai": "^2.12.0",
+    "sinon-chai": "^2.14.0",
     "sprintf-js": "^1.0.3",
     "style-loader": "^0.14.0",
-    "tether": "^1.3.2",
+    "tether": "^1.4.3",
     "typeahead.js": "^0.11.1",
-    "vue": "^2.4.2",
+    "vue": "^2.5.11",
     "vue-loader": "^11.1.3",
     "vue-resource": "^1.3.4",
-    "vue-router": "^2.7.0",
-    "vue-template-compiler": "^2.4.2",
-    "vuex": "^2.1.2",
+    "vue-router": "^2.8.1",
+    "vue-template-compiler": "^2.5.11",
+    "vuex": "^2.5.0",
     "webpack": "^2.7.0",
-    "webpack-dev-server": "^2.6.1"
+    "webpack-dev-server": "^2.9.7"
+  },
+  "dependencies": {
+    "url-polyfill": "^1.0.10"
```

Comment:
@eternoendless I have a question here: URL() is not available on IE, how can I use the polyfill in a conditional way?